src/qibotn/backends/quimb.py

```diff
@@ -24,12 +24,40 @@ def __init__(self, runcard):
             else:
                 raise TypeError("MPS_enabled has an unexpected type")
 
+            global TEBD_enabled
+            global TEBD_option
+            TEBD_enabled = runcard.get("TEBD_enabled")
```

Comment:
This variable is internal, no need to match 1-to-1 the option name. Thus, just follow the conventional naming rules
```suggestion
            tebd_enabled = runcard.get("TEBD_enabled")
```